app/models/benefitting_country.rb

```diff
@@ -1,16 +1,24 @@
 class BenefittingCountry
   include ActiveModel::Model
-  attr_accessor :name, :recipient_code, :code, :regions
+  attr_accessor :name, :recipient_code, :code, :graduated, :regions
 
   class << self
     def all
       @all ||= Codelist.new(type: "benefitting_countries", source: "beis").map { |country| new_from_hash(country) }
     end
 
+    def non_graduated
+      @non_graduated ||= all.select { |country| !country.graduated }
```

Comment:
Could this be more snappily expressed as
```
@non_graduated ||= all.reject(&:graduated)
```
?

Reply:
Yes I think it could!!